### ISSUE TITLE
Android: Rotating the Device Prompts the Unsupported Block Editor to Refresh

### ISSUE THREAD

**Issue by SiobhyB** [wordpress-mobile/gutenberg-mobile#3569]
**Describe the bug**

While editing content in the unsupported block editor on an Android device, a device rotation will cause the screen to refresh and any edits to be lost.

**To Reproduce**

_**_Precondition:_** On a WordPress.com or Jetpack-connected site, add a post via the web, then add a block that's not yet supported on mobile (e.g. Embed at time of writing)._

1. Within the Android app, navigate to a post that has a block that's currently unsupported on mobile. 
2. Tap on the unsupported block and then choose the option to _Edit using web editor_.
3. Begin editing the block's content within the newly loaded web editor.
4. After making edits, rotate the device so that the app's orientation changes.

**Expected behavior**

I would expect edits to survive a screen rotation, but they're lost. 

**Screenshots**

You'll see from the following video, that the tag cloud block (currently unsupported on mobile) is refreshed with the screen rotation:

https://user-images.githubusercontent.com/2998162/120319216-af9da200-c2d8-11eb-9540-0b643bfeb2a9.mov

**Smartphone (please complete the following information):**
 - Device: Pixel 5
 - OS: Android version 11
 - Version: alpha-296

**Comment by SiobhyB** [comment]
@mchowning, I don't think I made it clear enough in the issue's description, but this issue can lead to content loss, as any unsaved content added to the UBE will be lost if the user simply rotates their device while editing. If a significant amount of content is added then this has the potential for being quite a big pain point. Do you think this is perhaps worth a higher priority with that consideration?

**Comment by mchowning** [comment]
> Do you think this is perhaps worth a higher priority with that consideration?

Makes sense to bump it up. I updated it to a medium, but feel free to go ahead and bump it to a high if you think that makes more sense.

**Comment by SiobhyB** [comment]
`Medium` seems right to me too, thank you!
